### ISSUE TITLE
Experiment review only shows the latest reported metrics that were sent by send_metrics.

### ISSUE THREAD

**Issue by wematan** [polyaxon/polyaxon#198]
### Describe the bug
Experiment review only shows the latest reported metrics that were sent by send_metrics.
You have to report all metrics in one call to send_metrics() In order to display all metrics together.
For example if one has 3 different metrics that are reported at different times/iterations in a training:
Accuracy - every 10k iterations,
Train Loss - every 1k iterations,
Test Loss - every 1k iterations,
Currently if i try to report each metric by itself at the time it was measured in the training the experiment review page in the web ui will display only the last reported metric and not all metrics.
if i would like to report all of them together i would have to wait every 10k iterations until i have all metric data.

### To Reproduce
for example:
```python
if iter % 1000 == 0: 
    curr_test_loss = calc_test_loss()
    curr_train_loss = calc_train_loss()
    send_metrics(test_loss=curr_test_loss, train_loss=curr_train_loss)
if iter % 10000 == 0:
    curr_accuracy = calc_accuracy()
    send_metrics(accuracy=curr_accuracy)
```
if i would like to have all metrics visible at all time in the experiment view i would need to use:
```python
send_metrics(accuracy=curr_accuracy, test_loss=curr_test_loss, train_loss=curr_train_loss)
```

### Expected behavior
When using send_metrics polyaxon should collect the metrics reported for each category and display it regardless of other metric category have been updated.
for example: the code above for reproducing the issue should enable a user to report all metrics at point of calculation while preserving other metrics that were already reported earlier.

**Comment by mouradmourafiq** [comment]
Closing this, it should be part of the next release (v0.2.7). There will be a seamless data migration to migrate previous experiments.
